src/components/DetailsPage/DetailsPage.tsx

```diff
@@ -127,12 +130,18 @@ const DetailsPage: React.FC<React.PropsWithChildren<DetailsPageProps>> = ({
           <BreadCrumbs data-test="details__breadcrumbs" breadcrumbs={breadcrumbs} />
         )}
         {showBackToRelease ? (
-          <a onClick={() => navigate(-1)} className="pf-c-button pf-m-link">
+          <Link
+            to={APPLICATION_RELEASE_LIST_PATH.createPath({
+              workspaceName: namespace,
+              applicationName,
+            })}
```

Comment:
This is very specific to release list page, `navigate(-1)` is correct. So if you are adding a onClick callback you should be using a button component, if you want to use `href` you should use a`Link` component.